### PR TITLE
Use specific version instead of branch name

### PR DESCRIPTION
**Pull request by andrzejewsky**
### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `data-test-id` are added for new elements
4. [ ] The changes are tested in Chrome/Firefox/Safari browsers and in light/dark mode
5. [ ] Your code works with the latest stable version of the core
6. [ ] I added changesets and [read good practices](../.changeset/README.md)

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] app
3. [ ] attribute
4. [ ] category
5. [ ] collection
6. [ ] customer
7. [ ] giftCard
8. [ ] homePage
9. [ ] login
10. [ ] menuNavigation
11. [ ] navigation
12. [ ] orders
13. [ ] pages
14. [ ] payments
15. [ ] permissions
16. [ ] plugins
17. [ ] productType
18. [ ] products
19. [ ] sales
20. [ ] shipping
21. [ ] translations
22. [ ] variants
23. [ ] vouchers

CONTAINERS=2

### CODE REVIEW

**Review by Droniu — APPROVED** [review]

**Review by poulch — APPROVED** [review]

File: .github/workflows/prepare-release.yml [hunk 1]
```diff
@@ -81,12 +81,13 @@ jobs:
           export GITHUB_TOKEN=$( \
             curl --request GET --url ${{ secrets.VAULT_URL}} --header "Authorization: JWT ${{ secrets.VAULT_JWT }}" | jq -r .token \
           )
+          CURRENT_VERSION=$(cat package.json | jq .version | xargs)
```
Comment (NyanKiyoshi) [inline .github/workflows/prepare-release.yml:84]: I don't see the reason for using `xargs` here? 
```suggestion
          CURRENT_VERSION=$(cat package.json | jq -r .version)
```

File: .github/workflows/prepare-release.yml [hunk 2]
```diff
@@ -81,12 +81,13 @@ jobs:
...
           gh api /repos/saleor/saleor-cloud-deployments/dispatches \
             --input - <<< '{
               "event_type": "open-release-pull-request",
               "client_payload": {
                 "project": "DASHBOARD",
                 "environment": "SANDBOX",
-                "version": "${{ github.ref_name }}"
+                "version": "$CURRENT_VERSION"
               }
             }'
```
Comment (NyanKiyoshi) [inline .github/workflows/prepare-release.yml:93]: Use `jq` to build JSON using variables:

```
$ jq --null-input --arg version 1.2.3 '
{
  "event_type": "open-release-pull-request",
  "client_payload": {
    "project": "DASHBOARD",
    "environment": "SANDBOX",
    "version": $version
  }
}
'
{
  "event_type": "open-release-pull-request",
  "client_payload": {
    "project": "DASHBOARD",
    "environment": "SANDBOX",
    "version": "1.2.3"
  }
}
```

**Review by NyanKiyoshi — CHANGES_REQUESTED** [review]

File: .github/workflows/prepare-release.yml (hunk 1 above)
Comment (andrzejewsky) [reply .github/workflows/prepare-release.yml:84]: to cut of the double quotes, will test that how it works without it

File: .github/workflows/prepare-release.yml [hunk 3]
```diff
@@ -81,12 +81,15 @@ jobs:
...
-              "client_payload": {
-                "project": "DASHBOARD",
-                "environment": "SANDBOX",
-                "version": "${{ github.ref_name }}"
-              }
-            }'
\ No newline at end of file
+
+          version=$(cat package.json | jq .version | xargs)
+          payload=$(jq --null-input --arg version $version '{
```
Comment (NyanKiyoshi) [inline .github/workflows/prepare-release.yml:86]:
```suggestion
          payload=$(jq --null-input --arg version "$version" '{
```

File: .github/workflows/prepare-release.yml [hunk 4]
```diff
@@ -81,12 +81,15 @@ jobs:
...
-              "event_type": "open-release-pull-request",
-              "client_payload": {
-                "project": "DASHBOARD",
-                "environment": "SANDBOX",
-                "version": "${{ github.ref_name }}"
-              }
-            }'
\ No newline at end of file
+
+          version=$(cat package.json | jq .version | xargs)
```
Comment (NyanKiyoshi) [inline .github/workflows/prepare-release.yml:85]:
```suggestion
          version=$(cat package.json | jq -r .version)
```

**Pull request merged**